agbenchmark/challenges/code/d3/data.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "TestCreateSimpleWebServer",
+  "category": ["code"],
+  "task": "Build a basic web server that responds to a GET localhost:8079/health with a 200 OK. Deploy this web server locally at the port 8079. ",
+  "dependencies": [],
+  "ground": {
+    "answer": null,
+    "should_contain": [],
+    "should_not_contain": [],
+    "files": [],
+    "type": "custom_python"
+  },
+  "mock": {
+    "mock_func": null,
+    "mock_task": null
+  },
+  "info": {
+    "difficulty": "basic",
+    "description": "Tests ability for the agent to build a simple web server locally",
+    "side_effects": ["tests if there is in fact an LLM attached"]
```

Comment:
this side effect is already tested by write to file, just leave the array blank. also difficulty is not basic, its probably medium. It looks like I'm nitpicking here but this is source of truth, takes minimal time to make this data.json have all required info